backend/src/Request/SendNotificationRequest.php

```diff
@@ -12,6 +12,8 @@ class SendNotificationRequest
 
```

Comment:
it will be better if we have single request for each notification type

Reply:
will we speak in meeting